a3/terraform/modules/cluster/gke/variables.tf

```diff
@@ -100,7 +100,7 @@ variable "host_maintenance_interval" {
   type        = string
   default     = "PERIODIC"
   validation {
-    condition     = contains(["PERIODIC"], var.host_maintenance_interval)
+    condition     = var.host_maintenance_interval == null || contains(["PERIODIC"], coalesce(var.host_maintenance_interval, "PERIODIC"))
```

Comment:
this might be a little simpler with:
```terraform
condition = var.host_maintenance_interval != null ? contains(
  ["PERIODIC"],
  var.host_maintenance_interval,
) : true
```

Reply:
updated.